### ISSUE TITLE
Annotation search in project and site

### ISSUE THREAD

**Issue by cofiem** [QutBioacoustics/baw-server#182]
from @jwim1: Make it possible to search for annotations by tag from projects and sites. This is probably best addressed with links to the annotation library that specify the project/site id.

See https://github.com/QutBioacoustics/baw-client/issues/190

**Comment by atruskie** [comment]
Agree, set, project/site id as QSP.

Simplest option is just a "search through this [project/site]" `<a />`

Or could be more complex and send search phrase through as QSP.

I like first option because it gives a browseable view straight up.

**Comment by cofiem** [comment]
This is blocked by the annotation library not loading the annotation search results. Is there a separate issue for that?

**Comment by atruskie** [comment]
https://github.com/QutBioacoustics/baw-client/issues/190

**Comment by cofiem** [comment]
How's the annotation library? :tongue:

**Comment by atruskie** [comment]
Blocked by auth issues. Blocked by deploying ap.exe. No further work done.

Its almost been refactored fully though

**Comment by cofiem** [comment]
Does the annotation library support filtering by projects/sites?

**Comment by atruskie** [comment]
Currently no. See https://github.com/QutBioacoustics/baw-client/issues/257 and #253
